### ISSUE TITLE
Lambda Debug Configuration Using JSON File as Payload Fails With "Invalid JSON" Error

### ISSUE THREAD

**Issue by joe-bethke** [aws/aws-toolkit-vscode#1439]
**Describe the bug**

I have tested this in versions 1.16.0 and 1.17.0 - **the error is new as of 1.17.0**, a workaround right now is to use extension version 1.16.0 (by right clicking the extension, and hitting 'install another version...'). This occurs using VSCode-insiders edition, I have not tested on the normal VSCode build yet.

When generating a debug configuration from an AWS-SAM defined lambda function via the AWS toolkit extension, the option to add a JSON payload to the debug lambda request is given, and providing a JSON file path is an option, like so:
```json
{
    "configurations": [
        {
            "type": "aws-sam",
            "request": "direct-invoke",
            "name": "Call Lambda",
            "invokeTarget": {
                "target": "code",
                "projectRoot": "my_project",
                "lambdaHandler": "app.myHandlerFunction"
            },
            "lambda": {
                "runtime": "nodejs12.x",
                "payload": {
                    "path": "path/to/my/payload/file"
                },
                "environmentVariables": {}
            }
        }
    ]
}
```

However, upon running this debug configuration, no matter what the `"path": "path/to/my/payload/file"` contains, a VSCode errors occurs saying:
```
Invalid JSON in payload file: path/to/my/payload/file
```

To reiterate, the contents of my payload file **are valid**, and I have tried several different file contents, and even an empty json object `{}`, but the error is still raised.

**To Reproduce**

1. Using the SAM CLI, create a new lambda function (hello-world example, for instance)
2. Create a debug configuration for your hello-world lambda function, by using the text editor annotation `add debug configuration` provided by the AWS Toolkit Extension
3. Add a JSON payload using the `"path"` parameter for your function's debug configuration (example above) in your `.vscode/launch.json` file
4. Attempt to run the debug configuration, start debugger
5. Error pops up

**Expected behavior**

The debug configuration should start as normal, and load the provided json file as the payload of the debug lambda request.

**Screenshots**

**Desktop (please complete the following information):**

-   OS: MacOS
-   Visual Studio Code Version: 1.53.0-Insiders
-   AWS Toolkit Version: 1.17.0

**Additional context**

**Comment by justinmk3** [comment]
Thanks for the report! A workaround is to specify an absolute path:

```
                "payload": {
                    "path": "/full/path/to/my/payload/file"
                },
```

The next Toolkit release will include https://github.com/aws/aws-toolkit-vscode/pull/1440 which  fixes this issue.

**Comment by justinmk3** [comment]
Fixed in 1.18
